tests/framework_starlette/test_bg_tasks.py

```diff
@@ -87,10 +87,10 @@ def _test():
         response = app.get("/" + route)
         assert response.status == 200
 
-    BUG_COMPLETELY_FIXED = (starlette_version >= (0, 21, 0)) or (
-        starlette_version >= (0, 20, 1) and sys.version_info[:2] > (3, 7)
+    BUG_COMPLETELY_FIXED = (0, 21, 0) <= starlette_version < (0, 23, 1) or (
```

Comment:
I'm so confused and it's not your fault. lol I think adding a comment here might be helpful:
```suggestion
    # The bug was fixed in version 0.21.0 but re-occured in 0.23.1.
    # The bug was also not present on 0.20.1 to 0.23.1 if using Python3.7.
    BUG_COMPLETELY_FIXED = (0, 21, 0) <= starlette_version < (0, 23, 1) or (
```

Reply:
Done!